shopify/packages/shopify/shopify_add_note.js

```diff
@@ -0,0 +1,86 @@
+// jshint esversion: 9
+
+/**
+ * @description null
+ * @param {ParamsType} params list of command parameters
+ * @param {?string} commandText text message
+ * @param {!object} [secrets = {}] list of secrets
+ * @return {Promise<SlackBodyType>} Response body
+ */
+
+const axios = require('axios');
+
+async function putRequest(url, request) {
+
+	return await axios.put(url, request)
+		.then((response) => { return response.data; })
+		.catch((error) => { return error.response.data; });
+}
+
+function isSecretMissing(secrets) {
+
+	var ret = '';

```

Comment:
```suggestion
	let ret = '';
```
Try to use `let` or `const` instead of `var` wherever possible.